jaxrs-api/src/main/java/module-info.java

```diff
@@ -22,6 +22,8 @@
     requires static jakarta.xml.bind;
 
     requires java.logging;
+    requires jakarta.cdi;
```

Comment:
Good point since it's not required on the client side. It opens on the question of what we should do with `jakarta.inject`. I'm okay with keeping that required as an implementation could use `@Inject` on the client. However, it would be a problem if CDI specific annotations are used.